docker-compose.yml

```diff
@@ -19,6 +19,7 @@ services:
       - BIO=Hey! Just a place where you can connect with me!
       - GITHUB=https://github.com/timothystewart6
       - TWITTER=https://twitter.com/TechnoTimLive
+      - MASTODON=https://mastodon.social/@Gargron
```

Comment:
Can you remove the example?  Keeping the example as is for now

Reply:
I made the mastodon as the twitter one: `MASTODON=https://mastodon.social/TechnoTimLive`.
Hope I did not miss understand you.